tests/config/install.sh

```diff
@@ -168,7 +168,7 @@ ARM="aarch64"
 OS="$(uname -m)"
 if [[ -n "$EXPORT_S3_STORAGE_POLICIES" ]]; then
     echo "$OS"
-    if [[ "$USE_DATABASE_REPLICATED" -eq 1 ]] || [[ "$OS" == "$ARM" ]]; then
```

Comment:
@kssenii : do you happen to know why azure is disabled for replicated database ?

Reply:
Probably a problem with ports https://github.com/ClickHouse/ClickHouse/pull/36171/commits/52832ea7882311f8caea58c39b94aaf9add08b56#diff-3575b7ff67cdd826106f096c6d432a807060a711e173ed06ced7e608c4b0c058R21-R25 when running run.sh.